### ISSUE TITLE
nvda does not read web sites when navigating microsoft edge chromium Using Mouse

### ISSUE THREAD

**Issue by gdata1** [nvaccess/nvda#9850]
Hello. Open any website
Try to navigate without a keyboard just a mouse.
nvda barely voices the text and objects on the site.
windows 10 18922 nvda alpha-17871,4672136a edge 77.0.204.0 canary and 77.0.197.1 dev

**Comment by gdata1** [comment]
also I noticed this problem in google chrome

**Comment by Adriani90** [comment]
Closing in favor of #10650, the discussion thee gets abit further.
